src/main/java/com/siftscience/ScoreRequest.java

```diff
@@ -45,6 +47,13 @@ protected HttpUrl path(HttpUrl baseUrl) {
             builder.addQueryParameter("abuse_types",
                 StringUtils.joinWithComma(scoreFieldSet.getAbuseTypes()));
         }
+        Set<String> fields = new HashSet<>();
+        if (scoreFieldSet.isReturnScorePercentiles()) {
+            fields.add("score_percentiles");
+        }
+        if (fields.size() > 0) {
+            builder.addQueryParameter("fields", StringUtils.joinWithComma(fields));
+        }
```

Comment:
Can be simplified, as fields may contain `score_percentiles` only
```suggestion
        if (scoreFieldSet.isReturnScorePercentiles()) {
            builder.addQueryParameter("fields", "score_percentiles");
        }
```